.github/workflows/perf-testing-setup.yml

```diff
@@ -2,35 +2,33 @@ name: Throughput testing workflow
 
 on:
   workflow_dispatch:
-  pull_request:
-    branches: [ dev ]
-
+    inputs:
+      profiling_sampling_rate:
+        description: 'Profiling sampling rate (tps)'
+        required: false
+        default: '500'
```

Comment:
Can you do the test_to_run as an argument as well?

scenario_to_run:
    description: Scenario to run. Options are - <few from the list>
    required: false
    default: 'SyncHelloWorld'